nixos/modules/services/backup/borgbackup.nix

```diff
@@ -250,6 +250,15 @@ in {
             '';
           };
 
+          persistent = mkOption {
+            type = types.bool;
+            description = ''
+              Set the <literal>persistent</literal> option for
+              the systemd-timer.
```

Comment:
`man`-pages can be referenced in docbook like this: `<citerefentry><refentrytitle>systemd.timer</refentrytitle><manvolnum>5</manvolnum></citerefentry>.`

Also please note that the man-page is named `systemd.timer` :)